src/frontend/src/components/common/ViewComment/index.tsx

```diff
@@ -0,0 +1,31 @@
+import CommentsSidebar from '../../public/CommentsSidebar';
+import { useContext, useEffect, useState } from 'react';
+import { PiaSections } from '../../../types/enums/pia-sections.enum';
+
+import ViewCommentProps from './interfaces';
+import {
+  IPiaFormContext,
+  PiaFormContext,
+} from '../../../contexts/PiaFormContext';
+
+const ViewComments = ({ comments, path, count }: ViewCommentProps) => {
+  const { piaCollapsibleChangeHandler, piaCommentPathHandler } =
+    useContext<IPiaFormContext>(PiaFormContext);
```

Comment:
Why props and context usage? 🤔🤔🤔

Reply:
the purpose of using context here is pass the path to parent component and parent component pass to comment sidebar component

Reply:
I just realize this ViewComment does not exist in the PIAForm component, it exists in each tab. Let's discuss tomorrow morning